### DISCUSSION TITLE
Is there groupfilter available in Microsoft connector

### DISCUSSION THREAD

**Discussion by jagadeesh-kancherla-tfs** [dexidp/dex#3383] [Q&A]
connectors:
  - type: microsoft
    id: microsoft
    name: Microsoft
    config:
      clientID: 
      clientSecret: 
      redirectURI: 
      tenant: 
 Currently we are using above config for microsoft 
 
 Previously we used ldap like below
      groupSearch:
         baseDN: dc=test,dc=com
         filter: "(&(objectCategory=Group)(name=K8S-*-*-*-*))"     
         
  Is there any similar approach we have available in Microsoft connector?

**Comment by nabokihms** [comment]
Hello, @jagadeesh-kancherla-tfs. There is the filter. You can find many info by following the link https://dexidp.io/docs/connectors/microsoft/#groups
